### DISCUSSION TITLE
截图后可否像QQ那样编辑？

### DISCUSSION THREAD

**Discussion by gbsix** [shuax/MouseInc.Settings#281] [Ideas]
截图后可否像QQ那样编辑，如添加箭头，文字，加框，马赛克？
如果不能的话，有没有办法截图后自动打开系统的画图并粘贴进去？

这个可能要求有点多，如果不能实现也可以理解。

**Comment by huoevl** [comment]
**[ [ "Snapshot", "Stick" ], [ "Snapshot", "ToClipboard" ] ]**
